area/area.go

```diff
@@ -126,18 +123,14 @@ func (m *GormAreaRepository) ListChildren(ctx context.Context, parentArea *Area)
 	return objs, nil
 }
 
-// ConvertToLtreeFormat converts data in UUID format to ltree format.
-func ConvertToLtreeFormat(uuid string) string {
-	//Ltree allows only "_" as a special character.
-	converted := strings.Replace(uuid, "-", "_", -1)
-	converted = strings.Replace(converted, pathSepInService, pathSepInDatabase, -1)
-	return converted
-}
-
-// ConvertFromLtreeFormat converts data to UUID format from ltree format.
-func ConvertFromLtreeFormat(uuid string) string {
-	// Ltree allows only "_" as a special character.
-	converted := strings.Replace(uuid, "_", "-", -1)
-	converted = strings.Replace(converted, pathSepInDatabase, pathSepInService, -1)
-	return converted
+// ToExpression returns a string in ltree format.
+// Joins UUIDs in the first argument using `.`
+// Second argument is converted and appended if needed
+func ToExpression(p path.LtreePath, this uuid.UUID) string {
```

Comment:
Yyes thanks @aslakknutsen , make sense to move to path.go
Now done in https://github.com/almighty/almighty-core/pull/873/commits/e7ece0edd6e22af0c82499c104e26e3a48833151